Specification.md

```diff
@@ -1779,6 +1866,110 @@ The `@context` field will be set to the ASCII string `fedi-e2ee:v1/api/revoke`.
 }
 ```
 
+#### POST api/totp/disenroll
+
+Purpose: Remove the current TOTP secret stored in the Public Key Directory for the current instance.
+
+An HTTP 200 OK request will contain the following response fields:
+
+| Request Parameter | Type   | Remarks                      |
+|-------------------|--------|------------------------------|
+| `@context`        | string | Domain separation            |
+| `current-time`    | string | [Timestamp](#timestamps)     |
+| `disenrollment`   | object | See next table               |
+| `signature`       | string | Signature of `disenrollment` |
+
+The `@context` field will be set to the ASCII string `fedi-e2ee:v1/api/totp/disenroll`.
+
+The disenrollment object will consist of the following fields:
+
+| Request Parameter | Type   | Remarks                                        |
+|-------------------|--------|------------------------------------------------|
+| `actor-id`        | string | The Actor ID that produced the signature.      |
+| `key-id`          | string | The specific public key to use for this actor. | 
+| `otp`             | string | One-Time Password (see [TOTP](#totp))          |
+
+**Example Response**:
+
+```json5
+{
+  "@context": "fedi-e2ee:v1/api/totp/disenroll",
+  "success": true,
+  "time": "1730909831",
+}
+```
+
```

Comment:
How should this fail?

Here is a possibility to figure out if otp was setup for this account.